lib/snapshot/detect_values.rb

```diff
@@ -0,0 +1,45 @@
+module Snapshot
+  class DetectValues
+    # This is needed as these are more complex default values
+    def self.set_additional_default_values
+      config = Snapshot.config
+
+      detect_projects
+
+      Snapshot.project = FastlaneCore::Project.new(config)
+      Snapshot.project.select_scheme
+
+      # Go into the project's folder
+      Dir.chdir(File.expand_path("..", Snapshot.project.path)) do
+        config.load_configuration_file(Snapshot.snapfile_name)
+      end
+
+      # Devices
+      unless config[:devices]
+        config[:devices] = Simulator.all.collect { |d| d.name }
```

Comment:
Prefer `map` over `collect`.<br>Pass `&:name` as an argument to `collect` instead of a block.